utils/swift_build_support/swift_build_support/products/swiftpm.py

```diff
@@ -101,8 +105,8 @@ def run_bootstrap_script(self, action, host_target, additional_params=[]):
             elif host_target != self.args.host_target:
                 helper_cmd += [host_target]
                 helper_cmd.append("--skip-cmake-bootstrap")
-                build_toolchain_path = self.install_toolchain_path(host_target,
-                                                                   force_local=True)
+                build_toolchain_path = self.get_install_destdir(
+                    self.args, host_target, self.build_dir) + self.args.install_prefix
```

Comment:
Now, only `get_install_destdir()` provides the path to the cross-compiled toolchain and its resource directory, so use that instead here.